docs/source/howto/queryable-encryption.rst

```diff
@@ -0,0 +1,176 @@
+            if app_label == "patientdata":
+                return db == "encrypted"
+            # Don't create other app's models in the encrypted database.
+            if db == "encrypted":
+                return False
+            return None
+
+        def kms_provider(self, model, **hints):
+            return "local"
+
+    DATABASE_ROUTERS = [EncryptedRouter()]
+
+You are now ready to use server-side :doc:`Queryable Encryption
+</topics/queryable-encryption>`.
+
+.. admonition:: KMS providers and credentials
+
+    The above example uses a local KMS provider with a randomly generated
+    key. In a production environment, you should use a secure KMS provider
+    such as AWS KMS, Azure Key Vault, or GCP KMS.
+
+    Please refer to :ref:`manual:qe-fundamentals-kms-providers`
+    for more information on configuring KMS providers and credentials as well as
+    :doc:`manual:core/queryable-encryption/fundamentals/keys-key-vaults`
+    for information on creating and managing data encryption keys.
+
+    You can also refer to the `Python Queryable Encryption Tutorial
+    <https://github.com/mongodb/docs/tree/adad2b1ae41ec81a6e5682842850030813adc1e5/source/includes/qe-tutorials/python>`_.
+
+.. _client-side-queryable-encryption:
+
+Client-side Queryable Encryption
+--------------------------------
+
+Queryable Encryption is considered "client-side" when the encrypted fields map
+is known at the time of connection to the database. This approach can be used
+when you have finished development and you have a fixed set of encrypted fields
+that you want to use in your production environment.
+
+Encrypted fields map
+~~~~~~~~~~~~~~~~~~~~
+
+In addition to the :ref:`settings described in the how-to guide
+<server-side-queryable-encryption-settings>` you will need to provide a
+``encrypted_fields_map`` to the ``AutoEncryptionOpts``.
+
+You can use the :djadmin:`showencryptedfieldsmap` management command to generate
+the schema map for your encrypted fields and then use the results in your
+settings::
+
+    python manage.py showencryptedfieldsmap --database=encrypted
+
+.. admonition:: Didn't work?
+
+    If you get the error ``Unknown command: 'createcachecollection'``, ensure
+    ``"django_mongodb_backend"`` is in your :setting:`INSTALLED_APPS` setting.
+
+Crypt shared library
+~~~~~~~~~~~~~~~~~~~~
+
+Additionally, you will need to ensure that the :ref:`crypt shared library is
+available <manual:qe-reference-shared-library>` to your Python environment.
+The crypt shared library is recommended over libmongocrypt for Queryable
+Encryption because it doesn't require an additional daemon process to run to
+facilitate Queryable Encryption operations.
+
+Settings
+~~~~~~~~
+
+Now include the generated schema map in your Django settings::
+
+    …
+    DATABASES["encrypted"] = {
+        …
+        "OPTIONS": {
+            "auto_encryption_opts": AutoEncryptionOpts(
+                …
```

Comment:
Should this include an example of how to specify the `crypt_shared` library path?

Reply:
Yes! Will add.

Reply:
done